iOS/MSMapsModules/MSMapsModules/MSGeoJsonParser/MSGeoJsonParser.m

```diff
@@ -0,0 +1,138 @@
+//
+//  MSGeoJsonParser.m
+//  MapsModulesSample
+//
+//  Created by Elizabeth Bartusiak (t-elbart) on 6/12/20.
+//  Copyright © 2020 Microsoft.
+//  Licensed under the MIT license.
+//
+
+#import "MSGeoJsonParser.h"
+#import <MicrosoftMaps/MicrosoftMaps.h>
+
+@implementation MSGeoJsonParser {
+  MSMapElementLayer *layer;
+};
+
++ (MSMapElementLayer *)parse:(NSString *)geojson {
```

Comment:
Consider having parse return an error object as an additional output parameter and nullable MSMapElementLayer in the return type. That's generally how objective C APIs signal errors instead of using exceptions.

Reply:
Would that mean that in other areas that throw an exception (e.g. when latitude is out of range), it should also return an error object instead of throwing an exception?

Reply:
Yes